packages/adblocker/src/request.ts

```diff
@@ -337,6 +335,7 @@ export default class Request<T = any | undefined> {
   public readonly id: string;
   public readonly tabId: number;
   public readonly url: string;
+  public readonly urlForMatching: string;
```

Comment:
I expect that you wanted to keep the same prefix, but I think it would be more natural to set it to `matchingUrl` rather than `urlForMatching`.